src/components/Loading.js

```diff
@@ -1,12 +1,15 @@
 import Penguin from "./penguin.gif";
+import cat1 from "./cat1.gif";
+import cat2 from "./cat2.gif";
+import cat3 from "./cat3.gif";
```

Comment:
Were these supposed to be used? If not, remove them.

Reply:
They were - but since @jsflca2012  didn't pull from dev for a while and the Loading component got updated, his updates got overwritten by the merge. He's going to need to re-write the cat loading component.